flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/reconciler/sessionjob/SessionJobReconciler.java

```diff
@@ -34,6 +34,7 @@
 import org.apache.flink.kubernetes.operator.utils.StatusRecorder;
 import org.apache.flink.runtime.messages.FlinkJobNotFoundException;
 import org.apache.flink.runtime.messages.FlinkJobTerminatedWithoutCancellationException;
+import org.apache.flink.util.ExceptionUtils.findThrowable;
```

Comment:
```suggestion
import static org.apache.flink.util.ExceptionUtils.findThrowable;
```

it's static import, and please run `mvn spotless:apply` after change.